### PR TITLE
Fix: correctly get whole body of commit messages

### PR DESCRIPTION
**Pull request by JPeer264**
When commits had more than one line it got separated as it was an own commit when `type: 'text'`.

```
Feat: my description

my body
```

got parsed to

```
["Feat: my description", "", "got parsed to"]
```

this fix makes it to

```
["Feat: my description\n\ngot parsed to"]
```

### CODE REVIEW

**Comment by JPeer264** [comment]
This is a copy of the other repo, but instead of 10 commits (as in the original repo) it is just reduced to 4 commits. Idk why there are so many objects, but we won't touch the plain git files anyway.

**Comment by JPeer264** [comment]
Maybe because I copied it and removed the last 6 commits. I could add a new repo to make it less files? But that depends on you, if you want me to do that :)

**Comment by aichbauer** [comment]
Do we need that many commits? We are only testing if the body message is copied correctly right? So we could simply add one or two commits to the existing repo fixture right?

**Comment by JPeer264** [comment]
I wanted to cover 4 cases. 

1. one liner
2. just one break after each other
3. multiple line breaks
4. a one liner after all line breaks (to check it won't break anything after)

**Comment by JPeer264** [comment]
better to have more tests than too less :D

File: test/fixtures/repo-body-commits/git/logs/HEAD [hunk 1]
```diff
@@ -0,0 +1,28 @@
+15be93c31ad87c9ced03ba0b60fc2fb55c977c5c 1fd8a8cf346d4d4c671aa343b2e93ad01f92df3c JPeer264 <jan.oster94@gmail.com> 1589302305 +0200	rebase -i (start): checkout 1fd8a8cf346d4d4c671aa343b2e93ad01f92df3c
```
Comment (aichbauer) [inline test/fixtures/repo-body-commits/git/logs/HEAD:1]: what about this file? This does not exist in the current repo right?

**Comment by aichbauer** [comment]
Yep better safe than sorry...

I just think that this fixture will bloat this repo, because it is currently really hard to get an overview of what we need, and now we have two of them. And I do not quite think that you just copied the other repo. Can you tell me exactly what you did? We should probably add the commits to the old repo, since we already make all the tests with the other one.

**Comment by aichbauer** [comment]
So I digged a bit deeper into it... We only keep stuff that is needed to get the commits, we do not keep the whole repo in the fixtures, because this will bloat this package repo. 

We do not need any `.js` files, we do not need the logs, etc...

Just add the commits to the old repo or use the new one, but remove all unnecessary stuff, just take a look at the current repo fixture to get an idea of what is really needed to test the commits.

**Comment by JPeer264** [comment]
Ya I will check what is not needed. I will not add the commits to the existing repo, as it will fail the existing test cases.

**Comment by JPeer264** [comment]
i removed now all unnecessary files

**Pull request merged**